app/components/c-modal.md

```diff
@@ -92,4 +92,43 @@
     },
   };
 </script>
+```
+
+### Multiple nested modals
+**Note**: For nested modals you need to define a "portal-target" for each level. This can be defined in the l-default component.
+
+`
+<portal-target name="modal-container-2" multiple />
+`
+
+```vue
+<template>
+  <div>
+    <e-button @click="exampleModalIsOpen = true">open 1st modal</e-button>
+    <c-modal :open="exampleModalIsOpen" @close="exampleModalIsOpen = false" title="I'm a modal!">
+      hello World..
```

Comment:
- [ ] Please add some more content for this modal so the stacking is better visible.